angular/projects/admin-nrpti/src/app/import/import.component.ts

```diff
@@ -119,6 +119,6 @@ export class ImportComponent implements OnInit {
   }
 
   postToApi(): Observable<any> {
-    return this.factoryService.startTask({ dataSource: 'epic' });
+    return this.factoryService.startTask({ dataSource: 'epic', recordType: 'inspection' });
```

Comment:
Added this here for now, as currently the import-task expects/requires exactly 1 dataSource and 1 recordType.